src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -3312,6 +3882,9 @@ public ModelingSubmission addModelingSubmissionWithEmptyResult(ModelingExercise
         return submission;
     }
 
+    /**
+     * Moved to {@link ModelingExerciseUtilService#addModelingSubmission(ModelingExercise, ModelingSubmission, String)}
+     */
     public ModelingSubmission addModelingSubmission(ModelingExercise exercise, ModelingSubmission submission, String login) {
```

Comment:
here as well, I am not sure if it maybe belongs in ParticipationUtilService instead